liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/applications/InstallAppMojoSupport.java

```diff
@@ -139,6 +140,15 @@ protected void installLooseConfigApp() throws Exception {
         config.toXmlFile(looseConfigFile);
     }
     
+    private boolean containsJavaSource(MavenProject proj) {       
+        File javaSourceDir = new File(proj.getBasedir(), "src/main/java");
```

Comment:
Can't this be configured to something else?  Can we get the value from the project descriptor?